.infra/index.ts

```diff
@@ -114,25 +114,25 @@ if (isPersonalizedDigestEnabled) {
   );
 }
 
-const memory = 640;
+const memory = 400;
 const limits: pulumi.Input<{
   [key: string]: pulumi.Input<string>;
 }> = {
-  cpu: '1',
+  cpu: '800m',
   memory: `${memory}Mi`,
 };
 
 const wsMemory = 2048;
 const wsLimits: pulumi.Input<{
   [key: string]: pulumi.Input<string>;
 }> = {
-  cpu: '500m',
+  cpu: '300m',
   memory: `${wsMemory}Mi`,
 };
 
 const bgLimits: pulumi.Input<{
   [key: string]: pulumi.Input<string>;
-}> = { cpu: '250m', memory: '256Mi' };
+}> = { cpu: '50m', memory: '150Mi' };
```

Comment:
Per graph `150Mi` seems like a tight fit?

Reply:
What happens if instance overshots?

Reply:
Nothing, this is just the minimum amount of resources it needs to run.

Reply:
Ah its a min, 👍

Reply:
When I say nothing, I mean in 99.9% of times. Those 0.1% times would occur if a k8s node was fully "used up" and causes "pressure", i.e. all resources are use, then a pod might get evicted, and rescheduled to another node, to relieve said pressure.